src/components/PlaylistCard/playlist-card.css

```diff
@@ -0,0 +1,81 @@
+.playlist-container {
+    display: flex;
+    flex-direction: column;
+    width: 30rem;
+    height: fit-content;
+    cursor: pointer;
+    margin: 1rem;
+}
+
+.playlist-container:hover {
+    opacity: 0.7;
+}
+
+.playlist-container.playlist-empty {
+    border: 1px solid var(--video-cta-color);
+}
+
+.playlist-empty-title {
+    text-align: center;
+    font-size: 1.3rem;
+}
+
+.playlist-container .playlist-img {
+    height: 15rem;
+    position: relative;
+}
+
+.playlist-container.playlist-empty .playlist-img {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
a util class you can make for these three properties, must be using at a lot of places